src/app/selectors.js

```diff
@@ -438,8 +438,7 @@ export const isConnecting = state => !!state.root.visitURL;
   @returns {String} The users userTag.
 */
 export const getActiveUserTag = state =>
-  state.root.controllerConnection &&
-  state.root.controllerConnection.info.user.identity;
+  state?.root?.controllerConnection?.info.user.identity;
```

Comment:
Ooooh.. optional chaining 🌟

Reply:
Too bad the output version of this is huge compared to the previous version :D

Reply:
Party pooper! 👎🏻